spec/drivers/mechanize_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+require 'shared_examples_for_api'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.